### DISCUSSION TITLE
Updating schema and inserting new table

### DISCUSSION THREAD

**Discussion by xmrhahnx** [ujjwalguptaofficial/JsStore#308] [Q&A]
Hi there, 
I'm trying to update the DB schema incrementing the version number to create a new table defined like so:

```
{
	name: "newtable",
	columns: {
		id:	{notNull: true, primaryKey: true, autoIncrement: true},
		field: 	{dataType: "string", notNull: true}
	}
}
```

Then I try to fill the new table with some values (not specifying the 'id' value, because autoIncrement is true):
`db.insert({into:"newtable", values:[{"field":"test"}]})`

but it returns me:
`{message: "Null value is not allowed for column 'id'", type: "null_value"}`

What am I doing wrong?
Thank you

**Comment by ujjwalguptaofficial** [comment]
you have specified notNull in autoincrement column, just remove `notNull` constraints and all should be good.

**Reply by ujjwalguptaofficial** [reply]
I will try to fix this in some time, as this is Ok to provide this constraint.
